bin/saya/saya.sh

```
@@ -0,0 +1,174 @@
+    --rpc-url http://localhost:5050 \
+    --registry $SAYA_FACT_REGISTRY \
+    --settlement-contract $SAYA_PILTOVER_ADDRESS \
+    --world $SAYA_WORLD_ADDRESS \
+    --prover-url $SAYA_PROVER_URL \
+    --store-proofs \
+    --starknet-url $SAYA_SEPOLIA_ENDPOINT \
+    --signer-key $SAYA_SEPOLIA_PRIVATE_KEY \
+    --signer-address $SAYA_SEPOLIA_ACCOUNT_ADDRESS \
+    --private-key $SAYA_PROVER_KEY \
+    --batch-size 1 \
+    --start-block $(expr $SAYA_FORK_BLOCK_NUMBER + 1) \
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)